targets/targets.mk

```diff
@@ -8,6 +8,7 @@ $(eval $(call GluonTarget,bcm27xx,bcm2709))
 $(eval $(call GluonTarget,ipq40xx,generic))
 $(eval $(call GluonTarget,ipq40xx,mikrotik))
 $(eval $(call GluonTarget,ipq806x,generic))
+$(eval $(call GluonTarget,kirkwood,generic))
 $(eval $(call GluonTarget,lantiq,xrx200))
```

Comment:
Move to broken targets at the bottom of this file.

Reply:
Done.